python/labbox_ephys/import_to_workspace.py

```diff
@@ -75,32 +75,39 @@ def _import_le_sorting(sortings_subfeed: kp.Subfeed, le_sorting):
         }
     })
 
-def import_recording(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, recording_id: str):
+def random_id():
+    return str(uuid.uuid4())[-12:]
+
+def import_recording(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, recording_label: str):
+    recording_id = 'R-' + random_id()
     x = {
         'recordingId': recording_id,
-        'recordingLabel': recording_id,
-        'recordingPath': ka.store_object(recording.object(), basename=f'{recording_id}.json'),
+        'recordingLabel': recording_label,
+        'recordingPath': ka.store_object(recording.object(), basename=f'{recording_label}.json'),
         'recordingObject': recording.object(),
-        'description': f'Imported from Python: {recording_id}'
+        'description': f'Imported from Python: {recording_label}'
     }
     recordings_subfeed = feed.get_subfeed(dict(workspaceName=workspace_name, key='recordings'))
     _import_le_recording(recordings_subfeed, x)
+    return x
 
-def import_sorting(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, sorting: se.SortingExtractor, recording_id: str, sorting_id: str):
+def import_sorting(*, feed: kp.Feed, workspace_name: str, recording: se.RecordingExtractor, sorting: se.SortingExtractor, recording_id: str, sorting_label: str):
+    sorting_id = 'S-' + random_id()
```

Comment:
Similar for importing a sorting... the sorting ID is randomly generated (the sorting_label is what displays in the GUI)